### ISSUE TITLE
Force annotation of maturation proteins?

### ISSUE THREAD

**Issue by taltman** [nawrockie/dnaorg_annotate_genomes.pl#20]
Hi Eric,

I hope this finds you well. I tried running VADR on the SARS-CoV-2 reference genome from NCBI, and against a distantly-related CoV genome that we found:

https://github.com/ababaian/serratus/wiki/Fr4NK

Below please see a screenshot showing the FTR output file for Fr4NK (above), and SARS-CoV-2 (below):

![image](https://user-images.githubusercontent.com/2146012/87273073-fef29e00-c48c-11ea-9325-8fdd44df49a2.png)

I realize that because Fr4NK is so divergent from the SARS-CoV-2 VADR model, it probably doesn't bother calling the maturation proteins. It's important for us to isolate the domains of these maturation proteins, so that we can build trees from our new CoV genomes and determine species & genus boundaries, as explained by the ICTV:

https://talk.ictvonline.org/ictv-reports/ictv_9th_report/positive-sense-rna-viruses-2011/w/posrna_viruses/222/coronaviridae

Is there a way to tell VADR to call the maturation proteins, even if the polyprotein "fails"? Is there a way to get the raw output from the infernal / hmmer3 / blastx calls? Or should I just use the SARS-CoV-2 Pfam models on the polyproteins called by VADR? Thanks in advance for your help!

**Comment by nawrockie** [comment]
Hi Tomer, 

The reason you are not getting mat_peptide annotation is because the best-scoring model is NC_010438 which, unlike NC_045512, does not have mat_peptide annotation. If it did, then the .ftr file you are looking at would have mat_peptide annotation for your sequence.

Here's two ways you get could get around this:
1. Try to annotate the mat_peptides in NC_010438 and update the corona.minfo file with those annotated positions.
2. Make a model subset of the corona set of 55 models with only models that do have mat_peptide info and rerun your sequence using that. Those models are: 
```
AC_000192
NC_001451
NC_003045
NC_003436
NC_004718
NC_006213
NC_006577
NC_010800
NC_019843
NC_022103
NC_045512
```

Or just these two models (which are the only two Alphacoronaviruses in that list (NC_010438 is also Alpha))
```
NC_022103
NC_003436
```

Also, v-annotate.pl will keep the infernal/hmmer/blast output files if you use the --keep option. But only the best matching model is used to do the annotation so you'll only see output using the NC_010438 model. 

Let me know if you have further questions.

**Comment by taltman** [comment]
Hi Eric,

Thank you for your detailed reply. I now understand why the Fr4NK genome didn't have its maturation proteins annotated.

For expediency, we're currently using the Pfam CoV models to annotate all of the maturation proteins on polyprotein ORF 1ab as called by VADR. But after our current sprint to get things done, it would be nice to circle back and improve the corona.minfo model annotations. Where does the current annotation derive from, the RefSeq annotations? Is there a way to submit an improved annotation there, upstream?

**Comment by nawrockie** [comment]
@taltman : this is fixed in version 1.1.1 now available as the latest master commit. Thanks again for reporting this, please continue to report any bugs you find.
